scripts/deploy-federation.sh

```diff
@@ -51,7 +51,7 @@ NS=federation-system
 PUBLIC_NS=kube-multicluster-public
 IMAGE_NAME="${1:-}"
 
-LATEST_IMAGE_NAME=quay.io/kubernetes-multicluster/federation-v2:canary
+LATEST_IMAGE_NAME=quay.io/kubernetes-multicluster/federation-v2:latest
```

Comment:
We also need to update `scripts/deploy-federation-latest.sh`.

Reply:
ok I will add that on.